__tests__/app.test.js

```diff
@@ -320,4 +338,39 @@ describe("app", () => {
       });
     });
   });
+  describe("DELETE tests", () => {
+    describe("/api/comments/:comment_id", () => {
+      test("DELETE 204: should accept a valid delete request, and respond with a status code 204 and no content", () => {
+        return request(app)
+          .get("/api/comments")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.allComments.length).toBe(18);
```

Comment:
key should be clean and simple for sending data back e.g. body.comments not allComments